src/Movies/Movies.js

```diff
@@ -3,26 +3,27 @@ import PropTypes from 'prop-types'
 import Card from '../Card/Card'
 import '../scss/_Movies.scss'
 
-const Movies = ({ movies, showMovieDetails }) => {
+const Movies = ({ movies, formatAverageRating }) => {
 	const moviesList = movies.map(movie => {
 		return (
 			<Card
 				key={movie.id}
 				movieId={movie.id}
 				title={movie.title}
 				poster={movie.poster_path}
-				rating={movie.average_rating}
-				showMovieDetails={showMovieDetails}
+				rating={formatAverageRating(movie.average_rating)}
 			/>
 		)
 	})
 
 	return (
-		<section className='Movies-container'>
-			<h2>View All Movies</h2>
-			{moviesList}
-		</section>
-	)
+    <section className="Movies-container">
+      {movies.length === 0 && (
+        <h3>There are currently no movies to rate.</h3>
```

Comment:
This pops up for one second right before the page loads with all of the movies. Is that okay with you or should we look in to somehow hiding this? I don't know if that's even possible. Just an idea!